### PR TITLE
PMM-4692 Check for breaking API changes.

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-440/

**Comment by AlekSi** [comment]
@BupycHuk PTAL

File: api/qanpb/collector.proto [hunk 1]
```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package qan;
+package qan.v1beta1;
```
Comment (BupycHuk) [inline api/qanpb/collector.proto]: why did you change it?

Comment (AlekSi) [reply api/qanpb/collector.proto]: https://github.com/uber/prototool/blob/dev/docs/breaking.md#beta-vs-stable-packages

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-440/
